### DISCUSSION TITLE
Is there a "click location" plugin? I can't seem to find one. :/

### DISCUSSION THREAD

**Discussion by jmuchovej** [jspsych/jsPsych#2102] [Q&A]
Is there a click-location plugin? Much like `webgazer` gives you "gaze location", I have some current work where we want folks to use clicks to estimate the location of objects.

(I'm not opposed to writing a plugin, but wanted to know if I missed something when looking through existing plugins.)

**Comment by kurokida** [comment]
Hi, @jmuchovej 
A plugin I created may be useful. Please see [the psychophysics plugin](https://jspsychophysics.hes.kyushu-u.ac.jp/).

**Reply by kurokida** [reply]
Please specify the response_type as 'mouse'. See [the documentation](https://jspsychophysics.hes.kyushu-u.ac.jp/pluginParams/).
